src/components/AddBook.js

```diff
@@ -0,0 +1,16 @@
+const AddBook = () => (
+  <form action="">
+    <fieldset>
+      <legend>Add new Book</legend>
+      <label htmlFor="title">
+        <input type="text" id="title" name="title" placeholder="Book Title" />
+      </label>
+      <label htmlFor="author">
+        <input type="text" id="author" name="author" placeholder="Author" />
+      </label>
+      <button type="submit">Add Book</button>
+    </fieldset>
+  </form>
```

Comment:
- [ ] Hello @mnekx, as you are in react module you should know that's not how react works, react components are not just a simple form please read this [Article](https://www.digitalocean.com/community/tutorials/how-to-create-custom-components-in-react)

Reply:
I have read the [article](https://www.digitalocean.com/community/tutorials/how-to-create-custom-components-in-react) but I think  I am missing your point 😞, I would appreciate if you were kind enough to show me what exactly is wrong with the compontents.

Reply:
In the code I am trying to use functional components like so in the [article](https://www.digitalocean.com/community/tutorials/how-to-create-custom-components-in-react):
![image](https://user-images.githubusercontent.com/22321030/159911176-c88c181f-057e-4686-b129-28bc9c2429b5.png). Please help me figure this out 🤔

Reply:
- [ ] hello @mnekx, maybe I didn't explain it clearly you are not using `import React from'react'`.